lib/formatters/stringFormatter.mjs

```diff
@@ -185,6 +185,12 @@ function formatter(messages, source, cwd) {
 			result = result.slice(0, result.lastIndexOf(ruleString));
 		}
 
+		if (message.fix) {
+			const { range, text: fixText } = message.fix;
+
+			result += dim(` (fix: ${range[0]}-${range[1]} -> "${fixText}")`);
+		}
+
```

Comment:
This is not covered in #8344, can you remove that change from this PR and open a separate issue for it?
I fear people might be reliant on the current format to be stable.
I know [you are relying on that change for your test](https://github.com/stylelint/stylelint/pull/8473/files#diff-0104e2476d83a456a5e8add0acc8334030af22abad28e606b33f9d8094c04e5dR800) but that doesn't warrant altering the formatter's output.

Reply:
I understand your point, but I think it's natural to include the option in the output when supported by the CLI. 
I don't think a separate issue is needed.

Reply:
*If* there was consensus on that, we would still have to discuss the format of the addition, its position and select the formatters that would have to be changed.
I would expect verbose to be the more likely candidate to hold that information.

I still don't think it's not the way to go about it and it should have its own PR/issue.

Reply:
I also think we need to discuss the output format more, so we should separate the output change from this PR.

For example, some fixed output may need more than one line. In this case, the `string` formatter's output may be broken for those who expect one line output.

This PR just adds a flag to enable `computeEditInfo`, right?

Reply:
Thanks for the discussion.
Okay, I'll separate it from this PR.

Reply:
I've removed formatter changes and updated the test.
[Remove formatter changes](https://github.com/stylelint/stylelint/pull/8473/commits/6dd504703e525d13646849363c0964e729f0f6c6)